packages/docs/src/components/Pricing/PricingPlansList/index.tsx

```diff
@@ -1,111 +1,90 @@
-import React, { useState } from "react";
+import React, { useMemo, useState } from "react";
 import styles from "./styles.module.css";
-import pricing from "../PricingCard/pricing.module.css";
-import Button from "../../Button";
-import PricingCard from "../PricingCard";
-import { clsx } from "clsx";
 import { PricingProps } from "..";
+import PricingPeriodButton from "./PricingPeriodButton";
+import usePageType from "@site/src/hooks/usePageType";
+import { pricingIndividualData } from "../pricingIndividualData";
+import { pricingBusinessData } from "../pricingBusinessData";
+import clsx from "clsx";
+import PricingCard from "../PricingCard";
 
-const PricingPlansList = ({ handleBusiness, handleIndividual }: PricingProps) => {
-  const [isMonthly, setIsMonthly] = useState(true);
+export type PlanType = "FREE" | "PRO" | "TEAM" | "ENTERPRISE";
 
-  const individual = isMonthly ? (
-    <>
-      $19 <span className={pricing.plan__currency}>USD</span>
-      <p className={pricing.plan__price_second_line}> per seat/month (excl. VAT) </p>
-    </>
-  ) : (
-    <>
-      $190 <span className={pricing.plan__currency}>USD</span>
-      <p className={pricing.plan__price_second_line}> per seat/year (excl. VAT) </p>
-    </>
-  );
-  const business = isMonthly ? (
-    <>
-      $29 <span className={pricing.plan__currency}>USD</span>
-      <p className={pricing.plan__price_second_line}> per seat/month (excl. VAT) </p>
-    </>
-  ) : (
-    <>
-      $290 <span className={pricing.plan__currency}>USD</span>
-      <p className={pricing.plan__price_second_line}> per seat/year (excl. VAT) </p>
-    </>
-  );
+export interface FeatureProps {
+  label: string;
+  info: string;
```

Comment:
nit: this should be an optional param so you could remove all the empty `info: ""` in pricingBusinessData.tsx and pricingIndividualData.tsx. it's cleaner
```suggestion
  info?: string;
```